tests/lean/StxQuot.lean.expected.out

```diff
@@ -6,7 +6,7 @@ StxQuot.lean:8:12-8:13: error: unexpected token ')'; expected identifier or term
 "(«term_+_» <missing> \"+\" (num \"1\"))"
 "(«term_+_» <missing> \"+\" (num \"1\"))"
 "(«term_+_» (num \"1\") \"+\" (num \"1\"))"
-StxQuot.lean:19:15-19:16: error: unexpected token ']'; expected term
+"(«term[_]» \"[\" [`x._@.UnhygienicMain._hyg.1 \",\"] \"]\")"
```

Comment:
Let's change this to a different invalid syntax like `[x+]` in the test file

Reply:
done